site/_scss/blocks/_details.scss

```diff
@@ -19,13 +26,11 @@
     background-color: var(--color-blue-lighter);
     border: 1px solid  var(--color-blue-medium);
   }
-
 }
 
-.details__header {
+.details > summary > .details__header {
```

Comment:
Is this necessary? The naming scheme appears to be BEM, but AFAIK BEM tries to avoid this kind of hierarchical selector pattern. This comment also applies to other similar selector patterns in the PR.

Reply:
I agree that it's breaking BEM, but think it's worth overriding here as the `details` and `summary` elements have a very specific layout which I want to match.